util/cmake/SeqAnSimdUtility.cmake

```diff
@@ -306,6 +320,14 @@ macro(add_simd_platform_tests target)
         set(umesimd_compile_blacklist "avx512_knl;avx512_skx;avx512_cnl")
         set(seqansimd_compile_blacklist "avx512_knl;avx512_skx;avx512_cnl")
 
+        ## seqan-simd:
+        # avx2: clang 3.7.x fails test cases:
+        #       SimdVectorTestCommon_ShuffleConstant1 type parameter (un-)signed char __vector(32) FAILED
+        #       SimdVectorTestCommon_ShuffleConstant1 type parameter (un-)signed short __vector(16) FAILED
+        if (NOT (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 3.6) AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 3.8)
```

Comment:
Does this improve readability? Because, I also didn't add parenthesis around this:
```cmake
if (COMPILER_CLANG AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 4.0)
```